package.json

```diff
@@ -14,15 +14,15 @@
   },
   "dependencies": {
     "@fingerprintjs/fingerprintjs": "^3.3.6",
-    "@polkadot/api": "^9.11.3",
-    "@polkadot/api-contract": "^9.11.3",
+    "@polkadot/api": "9.13.6",
```

Comment:
Same again here, are we locking to polkadot versions?

Reply:
Yes :)